.ci/pull-request-config.yaml

```diff
@@ -3,7 +3,7 @@ version: "2.1"
 dependencies: ./project-dependencies.yaml
 
 build:
-  - project: kiegroup/drools
+  - project: apache/incubator-kie-drools
```

Comment:
Now changed from `kiegroup/drools` to `apache/incubator-kie-drools`.

Reply:
Hi @lampajr , this is a little irregular case. `drools-ansible-rulebook-integration` is developed in `kiegroup`, but it depends on the latest SNAPSHOT of `apache/incubator-kie-drools`. Do you think these yaml changes are valid?

Reply:
Thank you for reviewing, @lampajr !

Reply:
@tkobayas I think this change is ok. But keep in mind that the idea is that we will have a kiegroup/drools main branch in sync with apache/incubator-kie-drools at some point (that will be our midstream). So maybe later we can move back to kiegroup/drools when that branch is in place?

Reply:
@rgdoliveira I wasn't aware that we wanted to restore kiegroup/drools and keep it in sync with the apache project (and tbh I don't see why we should), but anyway are you ok with this change for now? If so can you please approve it so I could merge? Thank you.